mods/default/functions.lua

```diff
@@ -327,6 +327,57 @@ function default.register_fence(name, def)
 end
 
 
+--
+-- Fence rail registration helper
+--
+
+function default.register_fence_rail(name, def)
+	minetest.register_craft({
+		output = name .. " 16",
+		recipe = {
+			{  def.material,  def.material,  def.material },
+			{  def.material,  def.material,  def.material },
+		}
+	})
```

Comment:
I think vertical separation is clearer, your recipe looks like a horizontal door and is more likely to be in use somewhere else. My suggestion is:
```
+		recipe = {
+			{def.material,  def.material},
+			{"", ""},
+			{def.material,  def.material},
+		}
+	})
```
Also note, you have lots of random spaces inside the brackets, please only add 1 space after the commas.

Reply:
ah, ok.

Reply:
Your recipe is shown at the top of this comment, it's 3 across. What i show is my suggestion, as i have written: "My suggestion is:", i can't be any clearer ;)